otri/filtering/filters/statistics_filter.py

```diff
@@ -1,167 +1,178 @@
-from ..filter import Filter, Stream, Collection
-from typing import Sequence, Mapping, Dict
+from ..filter import Filter, Stream, Sequence, Any, Mapping
+from typing import Sequence, Mapping, Collection
 from numbers import Number
 
 
 class StatisticsFilter(Filter):
+    '''
+    Pops a single piece of data, reads the fields in the `keys` init parameter, updates the state
+    and outputs the data unmodified.
 
-    def __init__(self, input_stream: Stream, keys: Collection[str]):
+    Input:
+        Single stream.
+    Output:
+        Single stream.
+    '''
+
+    def __init__(self, input: str, output: str, keys: Collection[str]):
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Redefining built-in 'input'](https://app.codacy.com/gh/OTRI-Unipd/OTRI/pullRequest?prid=5649486)